tsdb/head_read.go

```diff
@@ -149,7 +149,29 @@ func (h *headIndexReader) SortedPostings(p index.Postings) index.Postings {
 	return index.NewListPostings(ep)
 }
 
+func (h *headIndexReader) ShardedPostings(p index.Postings, shardIndex, shardCount uint64) index.Postings {
+	out := make([]storage.SeriesRef, 0, 128)
+
+	for p.Next() {
+		s := h.head.series.getByID(chunks.HeadSeriesRef(p.At()))
+		if s == nil {
+			level.Debug(h.head.logger).Log("msg", "Looked up series not found")
```

Comment:
Is this useful as a log message?  I expect if I read it I would immediately want to know which series.
Suggest it would be just as useful as a counter.

Reply:
This mimics the behaviour already existing in `SortedPostings()`. Maybe we can address both separately in a follow up PR?